lib/kcl.js

```diff
@@ -221,7 +221,21 @@ module.exports = function(config, kinesis) {
 
       if (resp.data.Records && resp.data.Records.length > 0) {
```

Comment:
If there are no records in this pull sequence, but the shard iterator isn't `null`, do we also want to use the same timeout? It seems to be set to `2500` right now.